scripts/get

```diff
@@ -46,6 +46,17 @@ initOS() {
   esac
 }
 
+# runs the given command as root (detects if we are root already)
+runAsRoot() {
+  local CMD="$*"
+  
+  if ! whoami | egrep -q '^root$'; then
```

Comment:
Thx! Very good point! Didn't know that `$EUID` exists (`env` doesn't show it).

I changed the PR via 8eace38, please have a look